lib/MusicBrainz/Server/Data/Relationship.pm

```diff
@@ -230,6 +230,55 @@ sub _load_related_info {
     $self->load_entities(@rels);
 }
 
+sub load_credits
```

Comment:
`Data::` methods starting with `load_` typically load some data onto an entity object (i.e. fill some Moose attribute with data). Maybe `get_credits` is a bit clearer for indicating it's only for fetching data?